Sources/SwiftSyntax/generated/SyntaxRewriter.swift

```diff
@@ -7072,7 +7072,7 @@ open class SyntaxRewriter {
   public func rewrite(_ node: Syntax) -> Syntax {
     let rewritten = self.visit(node)
     return withExtendedLifetime(rewritten) {
-      return Syntax(node.data.replacingSelf(rewritten.raw, arena: SyntaxArena()))
+      return Syntax(node.data.replacingSelf(rewritten.raw, arena: rewritten.raw.arena))
```

Comment:
> /Users/ec2-user/jenkins/workspace/swift-syntax-PR-macOS/branch-release/5.9/swift-syntax/Sources/SwiftSyntax/SyntaxArena.swift:164: Fatal error: an arena can't have a new child once it's owned by other arenas

@ahoppen https://github.com/apple/swift-syntax/commit/72fc0984e22f09de591e7433f40ea01afc691b7a#diff-a32c2f4fb5bb11223905d6e201c2c67a595c64775f6d2b9463b478a797e194f9 looks like we forgot to change the generating file :(

Reply:
Oh, interesting. Apparently we don’t verify the generated source code in 5.9.

Reply:
Opened https://github.com/apple/swift-syntax/pull/1812